velocity/src/main/java/org/geysermc/floodgate/VelocityPlugin.java

```diff
@@ -61,6 +64,10 @@ public VelocityPlugin(ProxyServer server, Logger logger) {
                 .maximumSize(500)
                 .expireAfterAccess(50, TimeUnit.SECONDS)
                 .build();
+
+        this.commandManager = commandManager;
```

Comment:
Command manager is not used outside of this constructor, so adding it as a variable in the class isn't necessary.